### ISSUE TITLE
Package isn't working with Guzzle 6.x

### ISSUE THREAD

**Issue by dvlpp** [RemiCollin/GuzzleCache#4]
Don't know if it's a big step, but it would be great if this package was compatible with Guzzle 6.

**Comment by RemiCollin** [comment]
Yes, I'm aware of that. There are lot of breaking changes in Guzzle 6, so I don't know how much work is required to upgrade to 6.
